h2o-genmodel-extensions/xgboost/src/test/java/hex/genmodel/algos/xgboost/XGBoostJavaMojoModelTest.java

```diff
@@ -16,4 +22,31 @@ public void testObjFunction() { // make sure we have implementation for all supp
     }
   }
 
+  @Test
+  public void testPredictContributionsSerialization() throws Exception {
+    MojoReaderBackend readerBackend = MojoReaderBackendFactory.createReaderBackend(
+            XGBoostJavaMojoModelTest.class.getResource("xgboost_java.zip"),
+            MojoReaderBackendFactory.CachingStrategy.MEMORY);
+    XGBoostJavaMojoModel mojo = (XGBoostJavaMojoModel) MojoModel.load(readerBackend);
+    PredictContributions pc = mojo.makeContributionsPredictor();
+    assertNotNull(pc);
+    assertTrue(deserialize(serialize(pc)) instanceof PredictContributions);
```

Comment:
should we test other affected algos the same way?
should we have the (de)serialize() method somewhere where we can share them?